README.md

```diff
@@ -16,21 +18,21 @@ Via yarn:
 yarn add @arcadia-eng/utility-connect-react
 ```
 
-## Quick start
+# Quick Start
 
-#### accessToken
+The Utility Connect widget can be instantiated for one of two user flows: creating a new User (obtaining their credentials for the first time), or updating an existing User (changing their credentials, for example, if their utility portal rejected their credentials that were on-file). The user flow is selected with the `config.scope` parameter described in the component API Reference below.
 
-This is the token used to authenticate API requests. It will be passed to the `open()` function used to open the module. If you are integrating with this package you should have instructions on how to generate tokens for your client. Note that the type of `accessToken` needed to instantiate the component depends on the "user flow" i.e. `create` vs. `update`
+Note that instantiating the Utility Connect widget requires an OAuth access token scoped to your user in order to authenticate API requests. See [Creating OAuth Tokens](https://developers.arcadia.com/#operation/createOAuthToken) for instructions. Note that the type of token differs if you are creating a user or updating a user -- see `config.accessToken` in the component API Reference below for details.
```

Comment:
> the Utility Connect widget requires an OAuth access token scoped to your user

this is only true in the EDIT flow -- we don't have a user to scope the token to in the initial flow